src/scss/includes/direction.scss

```diff
@@ -335,14 +335,19 @@ button.direction_shortcut {
 
     .mobile-roadmap-preview-close {
       border: none;
-      padding: 0 0 0 $spacing-xs;
```

Comment:
I think now we should be able to use our standard `FloatingButton` component for that, and remove all this specific CSS.